### ISSUE TITLE
Docker fails to start after upgrade from 17.06.2-ce-win27 to 17.09.0-ce-win33

### ISSUE THREAD

**Issue by AdarWesley** [docker/for-win#1314]
### Expected behavior
Docker should start normally, with all images and containers in tact.

### Actual behavior
Upgrade from the completed successfully.  I.e. it showed the message upgrade completed successfully.
Whale did not show up in system tray, even after a restart.

Docker for Windows Service appears to be running.

MobyLynux VM is missing from Hyper-V Manager
The only Virtual Switch in Virtual Switch Manager window is: Layered_Ethernet

### Information
See attached zip file with all the information from my environment (since the tray icon didn't show, 
could not upload a diagnostic log).

[Docker status 17.09.0-ce-win33.zip](https://github.com/docker/for-win/files/1464110/Docker.status.17.09.0-ce-win33.zip)

PS C:\WINDOWS\system32> [System.Environment]::OSVersion.Version
Major  Minor  Build  Revision
-----  -----  -----  --------
10     0      15063  0

Tried to create the VM manually but could not get Docker to run.

Rolling back to version 17.06.2-ce-win27 after uninstalling docker does work again.

### Steps to reproduce the behavior
Update to the new 17.09.0-ce-win33 version

**Comment by AdarWesley** [comment]
And checked again with version: Version: 18.03.0-ce-win59 (16762)
and this still happens.

**Comment by AdarWesley** [comment]
So, the good news is that this issue is now resolved for me.  What happened is that 7 months have passed since I ran into this issue.  In the meantime, I stayed on version 17.06.2-ce-win27 which worked all this time.
Now I tried to upgrade again to the current latest version: 18.06.0-ce-win72 and this upgrade succeeded and everything is working nicely, including the K8S support.

What changed in the meantime is that OS got a few updates from MS, plus I got a couple of BIOS updates from Lenovo.  My current OS version is: Windows 10 Pro version 1803 (OS Build 17134.165)
Not sure what of all these changes actually resolved the issue.
